metadata-ingestion/README.md

```diff
@@ -7,14 +7,20 @@ This supports sending data to DataHub using Kafka or through the REST api.
 
 ![metadata ingestion framework layout](../docs/imgs/datahub-metadata-ingestion-framework.png)
 
-The architecture of this metadata ingestion framework is heavily inspired by Apache Gobblin (also originally a LinkedIn project!). We have a standardized format - the MetadataChangeEvent - and sources and sinks which respectively produce and consume these objects. The sources pull metadata from a variety of data systems, while the sinks are primarily for moving this metadata into DataHub.
+The architecture of this metadata ingestion framework is heavily inspired by [Apache Gobblin](https://gobblin.apache.org/) (also originally a LinkedIn project!). We have a standardized format - the MetadataChangeEvent - and sources and sinks which respectively produce and consume these objects. The sources pull metadata from a variety of data systems, while the sinks are primarily for moving this metadata into DataHub.
 
 ## Pre-Requisites
 
 Before running any metadata ingestion job, you should make sure that DataHub backend services are all running. If you are trying this out locally, the easiest way to do that is through [quickstart Docker images](../docker).
 
 <!-- You can run this ingestion framework by building from source or by running docker images. -->
 
+## Migrating from the old scripts
+If you were previously using the `mce_cli.py` tool to push metadata into DataHub: the new way for doing this is by creating a file source and pointing it to your JSON file, and then using the DataHub sink.
```

Comment:
Is the new way to "create a file source" or "configure the file source" and pointing it to your JSON file.

Reply:
Good catch! Users just need to configure the file source. I've updated the docs accordingly.